### ISSUE TITLE
[Feature request]better armor-value lock.

### ISSUE THREAD

**Issue by wootaboot** [Coders-After-Dark/Reskillable#170]
the armor-value lock feature does not take into account that different pieces of the same set have different values. i.e diamond armor boots and diamond armor chest would have very different requirements because chest gives more armor. just want to have a feature that takes that into consideration due to the randomness of armor making with TiC armor.

**Comment by pupnewfster** [comment]
Are you talking about the ability to lock armor or the integration that CompatSkills (while also done by us this is not the correct issue tracker for): https://docs.blamejared.com/1.12/en/#Mods/CompatSkills/Supports/Minecraft/Armor_Point_Locking/

**Comment by wootaboot** [comment]
yes, i set it so that every .8 of armor requires 1 level of defense. however this makes it so the best armor takes different levels to use where as i want it to have level 32 defense. normally i would just give the armor fixed values but with TiC armor add-on, the armor isnt fixed so there is no way for me to do it that way.(boots take level 12 but chest takes level 32). i fix would be a compatability patch for the armor add-on that works with defense instead of armor.(Defense is fixed and tells you how much armor you will have with a full set and each piece gives a percentage of the defense in armor.
![2019-09-25_18 52 06](https://user-images.githubusercontent.com/55807098/65645486-b0d7e100-dfc5-11e9-9495-a02a223f55b4.png)
![2019-09-25_18 51 55](https://user-images.githubusercontent.com/55807098/65645489-b59c9500-dfc5-11e9-8d7c-a8a72b6e1f6c.png)
